lib/room.cpp

```diff
@@ -3009,3 +3010,12 @@ bool MemberSorter::operator()(User* u1, QStringView u2name) const
 
     return n1.localeAwareCompare(n2) < 0;
 }
+
+void Room::activateEncryption()
+{
+    if(usesEncryption()) {
+        qCWarning(E2EE) << "This room is already encrypted";
```

Comment:
```suggestion
        qCWarning(E2EE) << "Room" << objectName() << "is already encrypted";
```
It's not always easy to discern to which room the log line pertains; better to identify it right in the log line.